### PR TITLE
Create node.js.yml

### PR DESCRIPTION
**Pull request by Annel18**
I have added the node.js.yml in the .github/workflows as Actions, it now passes the automated

npm ci
npm run build
npm test

### CODE REVIEW

**Comment by Annel18** [comment]
I have added the node.js.yml  in the .github/workflows as Actions,  it now passes the automated
- npm ci
- npm run build
- npm test

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
```
Comment (rasimsen) [inline .github/workflows/node.js.yml:1]: can you rename the file as a build.yml, please. it will be more descriptive.

File: package.json [hunk 2]
```diff
@@ -11,6 +11,8 @@
         "test": "jest"
     },
     "dependencies": {
+        "actions": "^1.3.0",
```
Comment (rasimsen) [inline package.json:14]: do you need it on prod? if not, please move it to dev!

File: package.json [hunk 3]
```diff
@@ -11,6 +11,8 @@
...
+        "ci": "^2.3.0",
```
Comment (rasimsen) [inline package.json:15]: do you need it on prod? if not, please move it to dev!

**Pull request merged**